docs/products/cassandra/howto/disable-cross-cluster-replication.md

```diff
@@ -61,15 +57,15 @@ your services for CCR purposes.
 
 1.  Log in to [Aiven Console](https://console.aiven.io/).
 
-2.  From the **Services** page, select an Aiven for Apache Cassandra
+1.  From the **Services** page, select an Aiven for Apache Cassandra
     service on which you'd like to enable CCR.
```

Comment:
This should say "to disable CCR" as this section is about disabling CCR by deleting services